config/docker-compose.yml.example

```diff
@@ -26,42 +26,50 @@ objdump:
   volumes:
     - /tmp:/tmp:ro
 
+passivetotal:
+  build: ../src/main/scala/org/holmesprocessing/totem/services/passivetotal
+  ports:
+    - "7720:8080"
```

Comment:
set port to 9710 please. Gives us space and I set all the no files to 97xx in a previous pull request